src/assets/scss/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$grey-btn-color: #c4c4c4;
+$active-grey-btn-color: #6A6A6A;
+$background-section-color: #FBFBFB;
+$nav-link-color: #494949;
+$link-hover-color: #000;
```

Comment:
zmień sobie nazwę na bardziej generyczną np. black (są różne podejścia), bo używasz to w kilku miejscach :)